### ISSUE TITLE
RaftFiveNodesTest crashes

### ISSUE THREAD

**Issue by ThorbenLindhauer** [zeebe-io/zeebe#1110]
Saw this today for the first time.

Maven says:

```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 07:58 min
[INFO] Finished at: 2018-07-31T14:12:45+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.20:test (default-test) on project zb-raft: There are test failures.
[ERROR]
[ERROR] Please refer to C:\camunda\repos\tngp\zeebe\raft\target\surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date]-jvmRun[N].dump, [date].dumpstream and [date]-jvmRun[N].dumpstream.
[ERROR] The forked VM terminated without properly saying goodbye. VM crash or System.exit called?
[ERROR] Command was cmd.exe /X /C ""C:\Program Files\Java\jdk1.8.0_172\jre\bin\java" -jar C:\camunda\repos\tngp\zeebe\raft\target\surefire\surefirebooter8867856789100836052.jar C:\camunda\repos\tngp\zeebe\raft\target\surefire 2018-07-31T14-05-00_528-jvmRun1 surefire5437178400878984091tmp surefire_103926127554896729494tmp"
[ERROR] Process Exit Code: 0
[ERROR] org.apache.maven.surefire.booter.SurefireBooterForkException: The forked VM terminated without properly saying goodbye. VM crash or System.exit called?
[ERROR] Command was cmd.exe /X /C ""C:\Program Files\Java\jdk1.8.0_172\jre\bin\java" -jar C:\camunda\repos\tngp\zeebe\raft\target\surefire\surefirebooter8867856789100836052.jar C:\camunda\repos\tngp\zeebe\raft\target\surefire 2018-07-31T14-05-00_528-jvmRun1 surefire5437178400878984091tmp surefire_103926127554896729494tmp"
[ERROR] Process Exit Code: 0
[ERROR]         at org.apache.maven.plugin.surefire.booterclient.ForkStarter.fork(ForkStarter.java:679)
[ERROR]         at org.apache.maven.plugin.surefire.booterclient.ForkStarter.fork(ForkStarter.java:533)
[ERROR]         at org.apache.maven.plugin.surefire.booterclient.ForkStarter.run(ForkStarter.java:279)
[ERROR]         at org.apache.maven.plugin.surefire.booterclient.ForkStarter.run(ForkStarter.java:243)
[ERROR]         at org.apache.maven.plugin.surefire.AbstractSurefireMojo.executeProvider(AbstractSurefireMojo.java:1077)
[ERROR]         at org.apache.maven.plugin.surefire.AbstractSurefireMojo.executeAfterPreconditionsChecked(AbstractSurefireMojo.java:907)
[ERROR]         at org.apache.maven.plugin.surefire.AbstractSurefireMojo.execute(AbstractSurefireMojo.java:785)
[ERROR]         at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:137)
[ERROR]         at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
[ERROR]         at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:154)
[ERROR]         at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:146)
[ERROR]         at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:117)
[ERROR]         at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:81)
[ERROR]         at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:56)
[ERROR]         at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
[ERROR]         at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:305)
[ERROR]         at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:192)
[ERROR]         at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:105)
[ERROR]         at org.apache.maven.cli.MavenCli.execute(MavenCli.java:956)
[ERROR]         at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:290)
[ERROR]         at org.apache.maven.cli.MavenCli.main(MavenCli.java:194)
[ERROR]         at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[ERROR]         at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[ERROR]         at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[ERROR]         at java.lang.reflect.Method.invoke(Method.java:498)
[ERROR]         at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
[ERROR]         at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
[ERROR]         at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
[ERROR]         at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
```

The test produced in two minutes a 300,000 lines log file with plenty of these messages:

```
14:10:21.691 [default-0-8005] [-zb-actors-0] DEBUG io.zeebe.raft - Got Append request in leader state 
14:10:21.691 [default-0-8005] [-zb-actors-0] DEBUG io.zeebe.raft - Got Append request in leader state 
14:10:21.691 [default-0-8005] [-zb-actors-0] DEBUG io.zeebe.raft - Got Append request in leader state 
14:10:21.691 [default-0-8005] [-zb-actors-0] DEBUG io.zeebe.raft - Got Append request in leader state 
```

[io.zeebe.raft.RaftFiveNodesTest-output.txt.zip](https://github.com/zeebe-io/zeebe/files/2245239/io.zeebe.raft.RaftFiveNodesTest-output.txt.zip)

**Comment by Zelldon** [comment]
I think is a dublicate of https://github.com/zeebe-io/zeebe/issues/1068

**Comment by Zelldon** [comment]
should be fixed with #1068
